Composer/packages/client/src/store/parsers/baseWorker.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import uniqueId from 'lodash/uniqueId';
+
+// Wrapper class
+export class BaseWorker<DispatchType> {
+  private worker: Worker;
+  private resolves = {};
+  private rejects = {};
+
+  constructor(worker: Worker) {
+    this.worker = worker;
+    this.worker.onmessage = this.handleMsg;
+  }
+
+  sendMsg = <Payload>(type: DispatchType, payload: Payload) => {
```

Comment:
So type is only used in the case of the lu worker. It makes more sense to make it a part of the payload instead of adding an argument to this method. 

Therefore we can remove the `DispatchType` generic because it will be part of the payload.